src/cmd/devdraw/cocoa-screen-metal.m

```diff
@@ -373,6 +373,13 @@ - (BOOL)applicationShouldTerminateAfterLastWindowClosed:(NSApplication *)theAppl
 	return YES;
 }
 
+- (void)windowDidResize:(NSNotification *)notification
+{
+	if(![myContent inLiveResize]) {
```

Comment:
I just realized that we probably need to check if img is nil before calling resizeimg.  It may get resized before attachscreen.